src/ui/hooks/recordings.ts

```diff
@@ -127,7 +129,7 @@ export async function getRecording(recordingId: RecordingId) {
 }
 
 export function useGetRecording(
-  recordingId: RecordingId | null
+  recordingId: RecordingId | null | undefined
```

Comment:
Why is the typing relaxed here?

Reply:
`getRecordingId` returns a `string` or `undefined` so I changed this to support either `null` or `undefined` since, afaict, there isn't a material difference for the GQL to accept either null-ish value.

Reply:
Alternatively, we could change `getRecordingId` to return `null` instead of `undefined`. /shrug